sdk/identity/identity/test/internal/node/managedIdentityCredential/msalMsiProvider.spec.ts

```diff
@@ -37,13 +37,28 @@ describe("ManagedIdentityCredential (MSAL)", function () {
       it("throws when both clientId and resourceId are provided", function () {
         assert.throws(
           () => new MsalMsiProvider("id", { resourceId: "id" }),
-          /provided at the same time./,
+          /At most, one of 'clientId', 'resourceId', or 'objectId' may be provided/,
```

Comment:
nit: consider a slightly shorter phrasing: "Only one of `clientId`, `resourceId`, or `objectId` can be provided"